### DISCUSSION TITLE
[Enhancement] Dark mode for anonaddy webapp

### DISCUSSION THREAD

**Discussion by Dyrimon** [anonaddy/anonaddy#360] [Feature Requests]
Since the secondary fonts(domain name, description etc.) are thin and barely decipherable in the white background, I think a darker mode will be much easier on the eyes. Hope this request will be considered.

**Comment by willbrowningme** [comment]
Thanks, this is on the roadmap - https://github.com/anonaddy/anonaddy/projects/1#card-54454194

I will likely coincide this with when I update the user interface in general using TailwindUI components.

**Comment by ghost** [comment]
Will dark mode get advantage of `prefers-color-scheme`?

**Comment by willbrowningme** [comment]
@lyrebird-sound-cloning-system Tailwind does support `prefers-color-scheme` - https://tailwindcss.com/docs/dark-mode so that shouldn't be a problem.

I haven't used the new tailwind dark mode yet so looking forward to trying it out.

**Comment by mannp** [comment]
Wondered if this one had progressed at all :) been a while and not sure how the road map priorities work :)

**Comment by willbrowningme** [comment]
Currently working on and prioritising the new user interface for the web application that will be a single page application.

Once I've done that (in the next few weeks) I will be able to add dark mode and other features that I've been putting off.

**Comment by mannp** [comment]
> Currently working on and prioritising the new user interface for the web application that will be a single page application.
> 
> Once I've done that (in the next few weeks) I will be able to add dark mode and other features that I've been putting off.

Hey @willbrowningme thanks for the quick response :) and status update, much appreciated 👍🏻 keep up the good work.

**Comment by codenyte** [comment]
Any updates?

**Comment by mannp** [comment]
`https://github.com/orgs/anonaddy/projects/1#`

Very happy to see an 'in progress' 👍🏻
